packages/user-interface/src/components/cases-list/cases-list.module.scss

```diff
@@ -20,6 +20,7 @@
 .cases__cards {
   display: flex;
   flex-direction: column;
+  --denhaag-card-width: 100%;
```

Comment:
could it be easier to make components by default 100%? For now it's harder to quick fix but for the future.

Reply:
Yeah, that would probably a good idea.